src/code/z_sram_NES.c

```diff
@@ -35,70 +35,90 @@ PersistentCycleSceneFlags sPersistentCycleSceneFlags[SCENE_MAX] = {
 #undef DEFINE_SCENE
 #undef DEFINE_SCENE_UNSET
 
-#define WEEK_FLG2(flg) (2 << (2 * BIT_FLAG_TO_SHIFT(flg)))
-#define WEEK_FLG(flg) (3 << (2 * BIT_FLAG_TO_SHIFT(flg)))
+// Each flag has 2 bits to store persistence over the three-day reset cycle
+// Only 1 of these bits need to be set to persist (Values 1, 2, 3).
+// Therefore, these two macros are functionally identical
```

Comment:
I think something like
```suggestion
// Therefore, the final game does not distinguish between these two macros in use
```
since they *can* be used differently, they just aren't.